pandas/core/frame.py

```diff
@@ -1292,7 +1302,13 @@ def __rmatmul__(self, other):
     # IO methods (to / from other formats)
 
     @classmethod
-    def from_dict(cls, data, orient="columns", dtype=None, columns=None) -> DataFrame:
+    def from_dict(
+        cls,
+        data,
```

Comment:
Can ``data`` be typed too?

Reply:
For sure!

(Not here though, in the interest of refining in the scope of the PR)

Reply:
yes these need to be smal / incremental PRs